presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationUtils.java

```diff
@@ -88,14 +88,14 @@ public static void updateCorrelationState(CorrelationState state, double x, doub
 
     public static double getCorrelation(CorrelationState state)
     {
-        // Math comes from ISO9075-2:2011(E) 10.9 General Rules 7 c x
```

Comment:
Btw in this case `state.getCount() == 0` should be handled separately to avoid `/ by zero`, which I guess may be the reason why the initial version was written that way.